### PR TITLE
Test selenium 1686

### PR DESCRIPTION
**Pull request by mpozhidaeva**

### CODE REVIEW

File: test/java/com/odysseusinc/arachne/portal/front/BaseTest.java [hunk 1]
```diff
@@ -32,32 +36,51 @@
...
+    protected static final Log logger = LogFactory.getLog(BaseTest.class);
+
     protected static final boolean USE_SSL = false;
+    protected static final int SOLR_PORT = 8985;
+    protected static final int PG_PORTAL_PORT = 5489;
+    protected static final int PG_DATANODE_PORT = 5490;
+    protected static final int MAIL_API_PORT = 8025;
+    protected static final int MAIL_SMTP_PORT = 1025;
+    protected static boolean LOCAL_RUNNING = false;
+    protected static String VERSION = "1.10.0-SNAPSHOT";
```
Comment (pavgra) [inline test/java/com/odysseusinc/arachne/portal/front/BaseTest.java]: rm

File: test/java/com/odysseusinc/arachne/portal/front/BaseTest.java [hunk 2]
```diff
@@ -100,38 +165,40 @@ public static void setup() {
...
                 .withEnv(portalEnvs)
                 .withNetwork(network)
                 .withExposedPorts(8080)
                 .waitingFor(arachneWaitStrategy);
 
         portalContainer.start();
-        String oldName = portalContainer.getContainerName().substring(1);
 
-        DockerClientFactory.instance().client().renameContainerCmd(oldName).withName(containerName).exec();
+        String oldName = portalContainer.getContainerName().substring(1);
```
Comment (pavgra) [inline test/java/com/odysseusinc/arachne/portal/front/BaseTest.java]: Looks like magic, not clear

substring(1);

File: test/java/com/odysseusinc/arachne/portal/front/BaseTest.java [hunk 3]
```diff
@@ -100,38 +165,40 @@ public static void setup() {
         portalEnvs.put("spring.mail.properties.mail.smtp.auth", "false");
         portalEnvs.put("spring.mail.properties.mail.smtp.starttls.enable", "false");
         portalEnvs.put("spring.mail.properties.mail.smtp.starttls.required", "false");
-        portalEnvs.put("portal.hostsWhiteList", "localhost," + containerName);
+        portalEnvs.put("portal.hostsWhiteList", "localhost," + portalContainerName);
 
         portalEnvs.put("jasypt.encryptor.password", System.getProperty("jasypt.encryptor.password"));
 
-        portalContainer = new GenericContainer("hub.arachnenetwork.com/portal:1.10.0-SNAPSHOT")
+        portalContainer = new GenericContainer("hub.arachnenetwork.com/portal:" + VERSION)
```
Comment (pavgra) [inline test/java/com/odysseusinc/arachne/portal/front/BaseTest.java]: const

File: test/java/com/odysseusinc/arachne/portal/front/BaseTest.java [hunk 4]
```diff
@@ -68,30 +91,72 @@
...
+        if (LOCAL_RUNNING) {
+            portalPostgresContainer = createLocalProfilePostgresContainer(portalPGContainerName, PG_PORTAL_PORT, "arachne_portal");
+            datanodePostgresContainer = createLocalProfilePostgresContainer(datanodePGContainerName, PG_DATANODE_PORT, "datanode");
+            solrContainer = createLocalProfileSolrContainer();
+
+            waitManualStarting(PORTAL_BASE_URL, "PORTAL");
+            waitManualStarting(DATA_NODE_BASE_URL, "DATANODE");
+        }
+
+        if (!LOCAL_RUNNING) {
```
Comment (pavgra) [inline test/java/com/odysseusinc/arachne/portal/front/BaseTest.java]: else

File: test/java/com/odysseusinc/arachne/portal/front/BaseTest.java [hunk 5]
```diff
@@ -141,8 +208,58 @@ public static void setup() {
...
+                                                                                     Integer mappedPort,
+                                                                                     String dbName) {
+
+        Map<String, String> envs = new HashMap<>();
+
+        envs.put("POSTGRES_USER", "ohdsi");
+        envs.put("POSTGRES_PASSWORD", System.getProperty("postgres.password"));
+        envs.put("POSTGRES_DB", dbName);
+
+        FixedHostPortGenericContainer container = new FixedHostPortGenericContainer("postgres:9.6.5");
```
Comment (pavgra) [inline test/java/com/odysseusinc/arachne/portal/front/BaseTest.java]: param? or const at least

File: test/java/com/odysseusinc/arachne/portal/front/BaseTest.java [hunk 6]
```diff
@@ -152,11 +269,17 @@ public static void shutdown() throws IOException {
...
-        datanodeContainer.stop();
-        portalContainer.stop();
+        if (!LOCAL_RUNNING) {
+            datanodeContainer.stop();
+            portalContainer.stop();
+        }
         mailhogContainer.stop();
 
-
+        if (LOCAL_RUNNING) {
```
Comment (pavgra) [inline test/java/com/odysseusinc/arachne/portal/front/BaseTest.java]: pls, else

File: test/java/com/odysseusinc/arachne/portal/front/BaseTest.java (hunk 2 above)
Comment (mpozhidaeva) [reply test/java/com/odysseusinc/arachne/portal/front/BaseTest.java]: I can add comment :)

**Comment by acumarav** [comment]
Legacy

**Pull request closed without merging**